deploy-service/common/src/main/java/com/pinterest/arcee/bean/AutoScalingUpdateBean.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2016 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.pinterest.arcee.bean;
+
+
+public class AutoScalingUpdateBean {
```

Comment:
Is it possible to reuse AwsVmBean? I think terminationPolicy can be added to AwsVmBean.